<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registration_Dialog</class>
 <widget class="QDialog" name="registration_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>331</width>
     <height>291</height>
    </rect>
   </property>
   <widget class="QComboBox" name="comboBox_courses">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>4</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLabel" name="label_fullname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Full name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_birthdate">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Birthdate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gender">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>230</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fullname">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_birthdate">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2025</year>
      <month>7</month>
      <day>20</day>
     </date>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_gender">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_reg_as_admin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>161</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Register as Administrator</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_registration">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_signUp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>SignUp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_registration_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icons/documentation/logos/signIn.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
